<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Praaline::Plugins::StatisticsPluginProsody::AnalyserMacroprosodyWidget</class>
 <widget class="QWidget" name="Praaline::Plugins::StatisticsPluginProsody::AnalyserMacroprosodyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabResultsTable">
      <attribute name="title">
       <string>Results Table</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutResults"/>
     </widget>
     <widget class="QWidget" name="tabCharts">
      <attribute name="title">
       <string>Charts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutCharts"/>
     </widget>
     <widget class="QWidget" name="tabMeasureDefinitions">
      <attribute name="title">
       <string>Measures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutMeasureDefinitions"/>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxStatisticalAnalysis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBoxProperties">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Analysis Options - Macro-units</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="3">
          <widget class="QCheckBox" name="checkBoxExcludePauses">
           <property name="text">
            <string>Exclude pauses</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="optionMUListFromFile">
           <property name="text">
            <string>Macro-units from File:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="comboBoxMUTier"/>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="optionMULIstFromTier">
           <property name="text">
            <string>Macro-units from Level:</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="editFilenameMUList"/>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="commandOpenFileMUList">
           <property name="text">
            <string>Open File...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Analysis Options - Corpus</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="labelCorpus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Corpus to analyse:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="CheckBoxList" name="comboBoxMetadataCom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="CheckBoxList" name="comboBoxMetadataSpk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="labelMetadataSpk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Include Metadata Spk:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBoxCorpus"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="labelMetadataCom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Include Metadata Com:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelMetadataAnnot">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Include Metadata Annot:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="CheckBoxList" name="comboBoxMetadataAnnot">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCommandLinkButton" name="commandAnalyse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Perform Analysis</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBoxList</class>
   <extends>QComboBox</extends>
   <header>pngui/widgets/CheckBoxList.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxCorpus</tabstop>
  <tabstop>comboBoxMetadataCom</tabstop>
  <tabstop>comboBoxMetadataSpk</tabstop>
  <tabstop>comboBoxMetadataAnnot</tabstop>
  <tabstop>optionMUListFromFile</tabstop>
  <tabstop>editFilenameMUList</tabstop>
  <tabstop>commandOpenFileMUList</tabstop>
  <tabstop>optionMULIstFromTier</tabstop>
  <tabstop>comboBoxMUTier</tabstop>
  <tabstop>checkBoxExcludePauses</tabstop>
  <tabstop>commandAnalyse</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
